inference-engine/thirdparty/clDNN/api/resample.hpp

```diff
@@ -54,13 +54,14 @@ enum class shape_calculation_mode : int32_t {
 enum class coordinate_transformation_mode : int32_t {
     /// @brief the coordinate in the original tensor axis `x` is calculated as `((x_resized + 0.5) / scale[x]) - 0.5`.
     half_pixel,
-    /// @brief the coordinate in the original tensor axis `x` is calculated by `(x_resized + 0.5) / scale[x] - 0.5 if out_shape[x] > 1 else 0.0`.
+    /// @brief the coordinate in the original tensor axis `x` is calculated by `(x_resized + 0.5) / scale[x] - 0.5 if output_shape[x] > 1 else 0.0`.
```

Comment:
It made a cpplint error.
/home/jade/work/openvino/inference-engine/thirdparty/clDNN/api/resample.hpp:63:  Lines should be <= 160 characters long  [whitespace/line_length] [2]
Do you have any other solution?

Reply:
* why `out_shape` is changed to `output_shape`?
* How did you run cpplint? I think there is no issue with original build script. Otherwise, it should have already reported from other CI instances.

Reply:
Done